### PR TITLE
Site: Add Jiajun Xie as committer

### PR DESCRIPTION
**Pull request by JiajunBernoulli**

### CODE REVIEW

File: site/_data/contributors.yml [hunk 1]
```diff
@@ -142,6 +142,11 @@
   githubId: jcamachor
   org: Hortonworks
   role: PMC
+- name: Jiajun Xie
```
Comment (libenchao) [inline site/_data/contributors.yml:145]: Would you mind to also add `pronouns` section like https://github.com/apache/calcite/blob/0dcf3ab4185647243255af29f0128b52737c1bf2/site/_data/contributors.yml#L51 ?

**Review by libenchao — COMMENTED** [review]
Looks great, I left one nit comment.

File: site/_data/contributors.yml (hunk 1 above)
Comment (JiajunBernoulli) [reply site/_data/contributors.yml:145]: Thanks for your review.

Most people didn't add `pronouns`, I prefer to be the same as most people.

**Review by libenchao — APPROVED** [review]
+1

**Pull request merged**